### ISSUE TITLE
Remove polarstrip habitable tiles

### ISSUE THREAD

**Issue by marko-pi** [longturn/freeciv21#2130]
If you switch off the single tile islands, there are still DOZENS of habitable single tiles attached to the South and North Pole continents. They are very unrealistic and annoying. I suggest removing them by a special option `/set polarstrip disabled` as in the web version. They could also be removed by default or limited to just a few (Antarctic Peninsula) as they are completely unrealistic in this large number.

![Untitled](https://github.com/longturn/freeciv21/assets/18025812/0bdebe77-fbd5-4850-9a14-688ef9639613)

**Comment by lmoureaux** [comment]
I counted 11 tiles in a map with xsize=36. The number of tiles attached to the poles seems lower when using `WRAPX|WRAPY` instead of the default `WRAPX` topology.

**Comment by marko-pi** [comment]
> I counted 11 tiles in a map with xsize=36. The number of tiles attached to the poles seems lower when using `WRAPX|WRAPY` instead of the default `WRAPX` topology.

I counted 10 (north) +7 (south) tiles on the default map size.  AI created 4 (north) + 5 (south) cities on those tiles.  This is just too much.

**Comment by marko-pi** [comment]
It is just unrealistic.  And it is kind of single tile islands, which can be turned off.

**Comment by lmoureaux** [comment]
What's unrealistic? That the "habitable" tiles are there (which is arguably just a different geography than the Earth) or that so many cities/bases have been built before the modern area?
I disagree that these are single tile islands, they're just habitable areas next to the ice sheet just like the Greenland coast.

**Comment by marko-pi** [comment]
Yes, there is Greenland (which is NOT an extension of the North Pole continent) and there is the Antarctic Peninsula. However, the number of people living on these and similar points is negligible. And when the AI uses these tiles, a few per cent of the total population can be found there! (The average population of these polar cities is whooping 3!)

Of course, if you think that geography does not need to resemble Earth (at all), then the current game similarity to Earth is exaggerated...

I do not think this discussion is useful. I have made a feature request. If you find it interesting, please implement it. If not, let me know so I do not expect it in vain.

**Comment by lmoureaux** [comment]
> I do not think this discussion is useful. I have made a feature request. If you find it interesting, please implement it. If not, let me know so I do not expect it in vain.

What I'm trying to understand is:
* What your true concerns are. The best solution might not be the one proposed by the user in the first place
* Reasons why I should prioritize this over the (currently) 292 other open issues

Your request is currently very low in my priority list.

> Of course, if you think that geography does not need to resemble Earth (at all), then the current game similarity to Earth is exaggerated...

Most maps don't resemble the Earth at all. Actually, the Earth scenarios aren't very good from the gaming perspective because they're highly unbalanced.

**Comment by marko-pi** [comment]
I guess I do not have a good argument to prioritise my feature request. At least it will not be rejected.
